spyder/plugins/explorer/tests/test_explorer.py

```diff
@@ -32,6 +43,15 @@ def project_explorer(qtbot):
     return widget
 
 
+@pytest.fixture
+def copy_path_file(qtbot):
```

Comment:
I suggest you call this fixture ``project_explorer_withfiles`` to be more clear about what it actually is,

Reply:
changed! Thanks

Reply:
:+1: